app/frontend/pairings/Pairings.svelte

```diff
@@ -0,0 +1,290 @@
+<script lang="ts">
+  import { onMount } from "svelte";
+  import Stage from "./Stage.svelte";
+  import { PairingsData, loadPairings } from "./PairingsData";
+  import { showIdentities } from "./ShowIdentities";
+  import FontAwesomeIcon from "../widgets/FontAwesomeIcon.svelte";
+  import ModalDialog from "../widgets/ModalDialog.svelte";
+  import { redirectRequest } from "../utils/network";
+  import GlobalMessages from "../utils/GlobalMessages.svelte";
+
+  let { tournamentId }: { tournamentId: number } = $props();
+
+  let data = $state(new PairingsData());
+  let showReportedPairings = $state(true);
+
+  onMount(async () => {
+    data = await loadPairings(tournamentId);
+  });
+
+  function addSwissStage(e: MouseEvent) {
+    e.preventDefault();
+
+    void redirectRequest(
+      `/tournaments/${tournamentId}/stages`,
+      "POST",
+      data.csrf_token,
```

Comment:
I'm surprised we have to pass this through props - is the CSRF token not already available through a meta tag? https://github.com/Null-Signal-Games/cobra/blob/main/app/views/layouts/application.html.slim#L6


(Screenshot from Cobra)

<img width="221" height="352" alt="Image" src="https://github.com/user-attachments/assets/dbcd308d-7377-4ce0-b09f-f414d7c930ca" />